bciers/apps/reporting/src/app/components/activities/uiSchemas/pulpAndPaperUiSchema.ts

```diff
@@ -0,0 +1,103 @@
+import FieldTemplate from "@bciers/components/form/fields/FieldTemplate";
+import CheckboxWidgetLeft from "@bciers/components/form/widgets/CheckboxWidgetLeft";
+import SourceTypeBoxTemplate from "@bciers/components/form/fields/SourceTypeBoxTemplate";
+import NestedArrayFieldTemplate from "@bciers/components/form/fields/NestedArrayFieldTemplate";
+import { InlineFieldTemplate } from "@bciers/components/form/fields";
+
+const uiSchema = {
+  "ui:FieldTemplate": FieldTemplate,
+  "ui:classNames": "form-heading-label",
+  pulpingAndChemicalRecovery: {
+    "ui:FieldTemplate": FieldTemplate,
+    "ui:widget": CheckboxWidgetLeft,
+    "ui:options": {
+      label: false,
+    },
+  },
+  sourceTypes: {
+    "ui:FieldTemplate": FieldTemplate,
+    "ui:options": {
+      label: false,
+    },
+    pulpingAndChemicalRecovery: {
+      "ui:FieldTemplate": SourceTypeBoxTemplate,
+      emissions: {
+        "ui:title": "Pulping and Chemical Recovery",
+        "ui:ArrayFieldTemplate": NestedArrayFieldTemplate,
+        "ui:FieldTemplate": FieldTemplate,
+        "ui:options": {
+          arrayAddLabel: "Add Pulping and Chemical Recovery",
+          label: false,
+          title: "Pulping and Chemical Recovery",
+          padding: "p-2",
+        },
+        items: {
+          "ui:order": [
+            "gasType",
+            "methodology",
+            "emission",
+            "equivalentEmission",
+            "carbonateName",
+            "purityOfCarbonate",
+            "annualAmount",
```

Comment:
```suggestion
            "carbonateName",
            "purityOfCarbonate",
            "annualAmount",
```
Remove these from the items order (they'll be adjacent to the array object)

Reply:
Pulled in 46fcf40.